aom/src/main/java/com/nedap/archie/aom/primitives/CTerminologyCode.java

```diff
@@ -63,17 +66,17 @@ public boolean isValidValue(TerminologyCode value) {
         if(getConstraint().isEmpty()) {
             return true;
         }
-        for(String constraint:getConstraint()) {
-            if(constraint.startsWith("at")) {
-                if(value.getCodeString() != null && value.getCodeString().equals(constraint)) {
-                    return true;
-                }
-            } else if (constraint.startsWith("ac")) {
-                if(value.getTerminologyId() != null && value.getTerminologyId().equals(constraint)) {
-                    return true;
-                }
-            }
+        if(value != null && value.getTerminologyId() != null && !value.getTerminologyId().equalsIgnoreCase("local") && !AOMUtils.isValueSetCode(value.getTerminologyId())) {
+            //this is a non-local terminology. If a term binding is there, we may be able to validate, if external, we wil not be able to
+            //TODO: check term bindings as well! However, no idea how to handle the URIs, as the value will not be URI. I think?
+            //so return true for now for non-local terminology values
+            return true;
```

Comment:
I'm not sure if we should return true if we don't support this at this moment. I think it is better to return false for now and only return true if we know the value actually matches the constraint.

Reply:
that means people cannot use the rm object validator if they rely on external terminology servers or term bindings at all. I consider that to be a problem.

We could make it configurable instead, perhaps?

Reply:
> We could make it configurable instead, perhaps?

That would be nice.